app/assets/stylesheets/sitewide/color-library.scss

```diff
@@ -49,3 +49,40 @@ $color-accent: $Veri-Peri;
 $color-highlight-urgent: $Crimson;
 $background-color: $color-light;
 $ranking-border-color: $color-light;
+// 
+
+
+
+:root {
+    --lightning-blue-hue-sat: 186, 68%;
```

Comment:
This stuff is a bit silly, just trying some stuff out to avoid relying on some scss funcs.